src/modules/atoms/link/link.mixin.pug

```diff
@@ -0,0 +1,6 @@
+mixin link(text, destination)
+  -
+    destination = destination || '#';
+    text = text || 'Click me!';
+
+  a(href=destination class="link")&attributes(attributes)= text
```

Comment:
you can use the class shortcut `.link` and don't need to put the class attribute in parenthesis